SU2_CFD/src/fluid/CNEMOGas.cpp

```diff
@@ -131,9 +131,48 @@ su2double CNEMOGas::ComputeGasConstant(){
   return GasConstant;
 }
 
-su2double CNEMOGas::GetrhoCvve() {
+su2double CNEMOGas::ComputeGamma(su2double *V){
 
-    Cvves = GetSpeciesCvVibEle();
```

Comment:
Here there are two ComputeGamma() functions at the moment:
ComputeGamma() and ComputeGamma(*V)

The plan is to move towards tracking Gamma as a numerics vector

Reply:
*numerics variable I assume, not vector (just clarifying for potential reviewers)

Reply:
Yes, well both.  It is stored in numerics as a VariableType

Reply:
where exactly ? I don't think I am following. I guess I am just confused about gamma being a vector or not. I suppose it should just be a single variable, but it seems like it's not... or it's "both" as you say. not sure what that means, pls clarify a bit more with context of the code :)